### PR TITLE
libimagtodo: Debug output

### LINKED ISSUE #628: imag-todo/libimagtodo: more debug output
We should add more debugging output in `libimagtodo` and also `imag-todo`.

### PR DESCRIPTION
**Pull request by matthiasbeyer**
(Waiting on merge of ~~#686~~ #692 )

Add some debug output in `libimagtodo`.

---

Closes #628

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
@mario-kr review pls!

File: libimagtodo/src/task.rs [hunk 1]
```diff
@@ -32,6 +34,8 @@ impl<'a> Task<'a> {
         r.read_line(&mut line);
         import_task(&line.as_str())
             .map_err_into(TodoErrorKind::ImportError)
+            .map_dbg_err_str("Error while importing task")
+            .map_err_dbg_trace()
```
Comment (mario-kr) [inline libimagtodo/src/task.rs]: how about a `map_err_dbg_trace_str()` function?

**Comment by mario-kr** [comment]
The only change is additional output via Result helper functions, I see no problems here.
It builds, so let's merge.
Also: Maybe leave such easy, low priority tasks for new contributors? The more so in imag core libs like store, link, and the like

File: libimagtodo/src/task.rs (hunk 1 above)
Comment (matthiasbeyer) [inline libimagtodo/src/task.rs]: Nah, that would be overkill IMHO. One has to draw a line somewhere.

**Comment by matthiasbeyer** [comment]
> Maybe leave such easy, low priority tasks for new contributors? 

Yes, definitively. But I kind of fear the git hook PR actually, so I keep myself busy with the small things :smile:

**Pull request merged**